Dalamud/Plugin/Internal/AutoUpdate/AutoUpdateManager.cs

```diff
@@ -236,6 +236,10 @@ private void OnUpdate(IFramework framework)
                         this.NotifyUpdatesAreAvailable(
                             this.GetAvailablePluginUpdates(
                                 DecideUpdateListingRestriction(behavior)));
+                        
+                        // Schedule the next update check opportunistically.
+                        this.nextUpdateCheckTime = DateTime.Now + TimeBetweenUpdateChecks;
```

Comment:
You should still schedule this inside the callback. That was supposed to prevent the case of someone starting the game with the notification open, going away and coming back, then clicking it off and immediately having another update - someone actually complained about that already, which is why I changed it. We should only calculate the next time from the point a notification is dismissed.